includes/payments/functions.php

```diff
@@ -1366,18 +1366,20 @@ function give_donation_amount( $donation_id = 0, $type = '' ) {
 /**
  * Get the amount associated with a payment
  *
- * @param int $payment_id Payment ID.
+ * @param int    $payment_id Payment ID.
+ * @param string $type       Define context of donation amount..
  *
  * @access public
  * @since  1.0
+ * @since  1.8.17 Added new argument $type.
  *
  * @return mixed
  */
-function give_get_payment_amount( $payment_id ) {
+function give_get_payment_amount( $payment_id, $type = 'donation' ) {
 
 	$payment = new Give_Payment( $payment_id );
 
-	return apply_filters( 'give_payment_amount', floatval( $payment->total ), $payment_id );
+	return apply_filters( 'give_payment_amount', floatval( $payment->total ), $payment_id, $type );
```

Comment:
@ravinderk We will have to change the number of argument wherever this filter was used in any of the Give add-ons to get rid out of `too few argument was passed` PHP error.

Reply:
@emgk As we discussed this does not need any backward compatibility